test/integration/multicluster/target3/testdata/slow-cooker.yml

```diff
@@ -0,0 +1,26 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: slow-cooker
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: slow-cooker
+  template:
+    metadata:
+      labels:
+        app: slow-cooker
+    spec:
+      containers:
+      - name: slow-cooker
+        image: buoyantio/slow_cooker:1.3.0
+        command:
+        - "/bin/sh"
+        args:
+        - "-c"
+        - |
+          sleep 15 # wait for pods to start
+          /slow_cooker/slow_cooker -metric-addr 0.0.0.0:9999 http://nginx:8080
```

Comment:
You can't use `http://nginx:8080` here, you'll have to use the fqn of an endpoint mirror. e.g `http://nginx-set-<no>-<target>.<namespace>.svc.cluster.local` where no is the instance number (e.g nginx-set-0) and target is the name of the target cluster.